server/jetstream_cluster.go

```diff
@@ -8379,7 +8380,7 @@ RETRY:
 	releaseSyncOutSem()
 
 	if n.GroupLeader() == _EMPTY_ {
-		return fmt.Errorf("catchup for stream '%s > %s' aborted, no leader", mset.account(), mset.name())
+		return fmt.Errorf("catchup for stream '%s > %s' aborted, no leader: %w", mset.account(), mset.name(), errCatchupAbortedNoLeader)
```

Comment:
Nitpick: the current error message will read:
`catchup for stream '%s > %s' aborted, no leader: catchup failed, no leader`

Reply:
changed so it now reads: `catchup aborted, no leader for stream '%s > %s'`